tables/atom.py

```diff
@@ -236,57 +225,75 @@ class Atom(metaclass=MetaAtom):
 
     .. rubric:: Atom attributes
 
-    .. attribute:: dflt
+    .. autoattribute:: dflt
```

Comment:
General comment:
I think that using `..auto*` primitives for attributes has the drawback that the attribute documentation is not available and visible on the interactive help system.
I'm not sure that we should go in that direction honestly, but I'm open to discuss.

Reply:
Undoing then, this has little to do with the type hints anyway.